### ISSUE TITLE
Improve search indexing

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#945]
On Google and Bing:
- remove part-up.com/start from index
- improve indexing of part-ups and tribes and swarm pages

**Comment by ralphboeije** [comment]
@wouterd did you have some thoughts on this issue?

**Comment by ralphboeije** [comment]
Found out today that the verification by google was failing, because of the change to https. I tried to verify it today using Google Analytics, but that won't work. It is verified somehow "by ownership transfer". The proposed verification method is adding a HTML to the homepage, but I don't know if that will work with Meteor? Otherwise, we can also add it to our domain registration in AWS Route53. Next actions are:
- [ ] Add robots.txt
- [ ] Add sitemap
